why-apollo-advantages-disadvantages-alternatives.md

```diff
@@ -23,25 +23,25 @@ summary = "It is not always simple to pick the right solution for a problem. In
 
 {{% read_before "This tutorial is part 2 of 2 in this series." "Part 1:" "Why GraphQL: Advantages, Disadvantages & Alternatives" "https://www.robinwieruch.de/why-graphql-advantages-disadvantages-alternatives" %}}
 
-It is not always simple to pick the right solution for a problem. In a GraphQL era where nothing is set in stone because it is an emerging technolgy, it's difficult to get the solutions for GraphQL powered JavaScript applications right. In the following sections, I want to give you insights why you should or should not use Apollo for using GraphQL in your JavaScript applications. In addition, it will show you alternatives in case you decide against Apollo. Even though I recommend to use Apollo for JavaScript and GraphQL applications, I want to mention not only the positive things about it, but also the negative aspects.
+Finding the right solution a given problem is not always simple, and web applications like GraphQL are a good example of how changing times make for constantly evolving challenges. Moreover, evolving challenges create a scenario where the solutions must also evolve, so even the number of choices becomes a task. This article will decipher the pros and cons of one such solution: Apollo for GraphQL, with alternative solutions in case you decide against it. 
 
-Before we dive into the advantages and disadvantages of Apollo, let's answer the following question first: What is Apollo? It seems to be Apollo's mission making GraphQL effortless in JavaScript applications. Whereas GraphQL is only the query language which has its reference implementation in JavaScript, Apollo builds its ecosystem on top of it to make GraphQL available in a simpler way for a broader audience. This includes the client-side as well as the server-side, because they provide a large ecosystem of libraries for both sides. They even provide you with an intermediate layer with Apollo Engine as GraphQL gateway. So basically if you are interested in using GraphQL in your JavaScript applications, Apollo with its popular ecosystem is a great choice.
+GraphQL is only the query language that has a reference implementation in JavaScript, and Apollo builds its ecosystem on top to make GraphQL available for a wider audience. This includes the client-side as well as the server-side, because they provide a large ecosystem of libraries for both. The libraries provide an intermediate layer, with the Apollo Engine as a GraphQL gateway. Essentially there's a reason Apollo is one of the most popular choices for using GraphQL in JavaScript applications
```

Comment:
"The libraries provide an intermediate layer," -> "The libraries provide an intermediate layer too,"